pkg/csprofiles/csprofiles.go

```diff
@@ -45,37 +43,6 @@ func GenerateDecisionFromProfile(Profile *csconfig.ProfileCfg, Alert *models.Ale
 		/*for the others, let's populate it from the alert and its source*/
 		decision.Value = new(string)
 		*decision.Value = *Alert.Source.Value
-
-		if strings.EqualFold(*decision.Scope, types.Ip) {
```

Comment:
isn't this suppression going to mess with scenarios that try to extend a ban decision to a range or such ?